python-backend/app/auth.py

```diff
@@ -66,7 +64,7 @@ def get_user_email():
 
 def get_current_user_security():
     rv = getattr(g, 'current_user_security', None)
-    if rv == None:
+    if rv is None:
```

Comment:
Usually a good practise . to not use string comparators with None

Reply:
I don't believe == is a string comparator?  == checks equivalence in terms of values, is checks equivalence in terms of identity (object reference). I suppose you can argue that None isn't a value, but we're not interested in object identity here.

Reply:
In a quick search "is None" seems to be preferred, so still a valid change.